class.eventcalendar.plugin.php

```diff
@@ -199,28 +200,33 @@ public function postController_beforeBodyInput_handler($sender) {
         $sender->addJsFile('eventcalendar.js', 'plugins/EventCalendar');
         $sender->addDefinition(
             'EventCalendarCategoryIDs',
-            json_encode(c('EventCalendar.CategoryIDs'))
+            json_encode(unserialize(c('EventCalendar.CategoryIDs')))
```

Comment:
Vanilla stopped the support for arrays in the config (somethig I still do not understand), but my prefered way now of dealing with such a simple list would be a comma separated list which gets exploded and imploded. Unserializing shouldn't be needed if serialization doesn't happen elsewhere. If it is done, that other place should be corrected